client/tunnel

```diff
@@ -0,0 +1,26 @@
+#!/bin/sh
+#
+if [[ $# -eq 0 ]] ; then
+    echo "Please provide the hostname:port as argument, for example $0 host_inside_vpc:3306"
+    exit 0
+fi
+
+
+REGION=eu-central-1
+NAME=bastion-ssm-iam
+DEV_LOCAL_PORT=$(( ( RANDOM % 60000 ) + 1024 ))
```

Comment:
How about one allows the User to override those variables:

```bash
region="${REGION:-eu-central-1}"
name="${NAME:-bastion-ssm-iam}"
random_port=$(( ( RANDOM % 60000 ) + 1024 ))
dev_local_port="${DEV_LOCAL_PORT:-$random_port}"
```

Reply:
good one.